lib/config/migration.ts

```diff
@@ -57,6 +59,12 @@ export function migrateConfig(
       'peerDependencies',
     ];
     const { migratePresets } = getGlobalConfig();
+    const migrations: Migration[] = [
+      new RequiredStatusChecksMigration(config, migratedConfig),
+    ];
```

Comment:
I think I'd put this in `migrations/index.ts`, with a little helper function that takes `config`, and `migratedConfig`. Once this list starts growing, if we migrate the migrations (!), it'll be easier to maintain IMO.

Reply:
Yes,please do that

Reply:
Done